src-tauri/src/capturer/mod.rs

```diff
@@ -80,98 +65,108 @@ pub async fn start_capture(app_handle: AppHandle) {
     // record multiple gifs without restarting the app
     *frames = Vec::new();
```

Comment:
I'm a little underconfident about this part. We call `start_recorder` first and then create a `Vec::new()` in `frames`. Are we sure this is the right order and works correctly? What happens if `start_recorder` has already captured a few frames by the time `Vec::new()` is called?

Reply:
afaik, recording will be taken care by recorder.get_next_frame() func.